### PR TITLE
PDF invoice generation

### PR DESCRIPTION
**Pull request by Lenetis**
Example invoice: https://bstrzelecki25.atlassian.net/jira/software/projects/SCRUM/boards/1?selectedIssue=SCRUM-12

### CODE REVIEW

File: test/invoice_generator_test.py
Comment (bstrzelecki) [inline test/invoice_generator_test.py:1]: missing test case for saving pdf

File: src/invoice_generator.py [hunk 1]
```diff
@@ -1,54 +1,112 @@
 import datetime
+import importlib.resources
 import uuid
-import xml.etree.cElementTree as ET
-from typing import Dict
+from dataclasses import dataclass, asdict
+from typing import Dict, List, Tuple
```
Comment (bstrzelecki) [inline src/invoice_generator.py]: any reason for using deprecated types?

Comment (Lenetis) [reply src/invoice_generator.py]: I didn't know they were deprecated. Changed to default dict, list, tuple

File: test/invoice_generator_test.py
Comment (Lenetis) [reply test/invoice_generator_test.py:1]: added test_generate_pdf()

File: src/invoice_generator.py [hunk 2]
```diff
@@ -1,54 +1,111 @@
...
+    client_name: str
+    client_surname: str
+    client_mail: str
+    client_subscription: str
+    client_subscription_cost: float
+    client_packages: list[tuple[str, float]]
+    total_cost: float
+    invoice_date: datetime.date
+    payment_due: datetime.date
+    company_name: str = "Drying Paint Inc."
```
Comment (bstrzelecki) [inline src/invoice_generator.py]: there is already merged pull request containing file that included all necessary constants (manifest.py)

**Review by bstrzelecki — COMMENTED** [review]
DeprecationWarning: PyPDF2 is deprecated. Please move to the pypdf library instead.

**Review by bstrzelecki — APPROVED** [review]

**Review by BognaLew — APPROVED** [review]

**Review by BognaLew — APPROVED** [review]

**Pull request merged**